### PR TITLE
Peer Review

### PR DESCRIPTION
**Pull request by jonhavbra87**
I would love to get an peer review from you guys! Here are the things I would love to hear about:

- User with @noroff.no or @stud.noroff.no email can register profile
- Registered user can login
- User can view a post content feed
- User can view a post content item by ID
- User can create a post content item
- User can update a post content item
- If you have time: design on the posts and posSpecific

Cheers!

### CODE REVIEW

File: js/api/auth/login.mjs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    body
+})
+
+const { accessToken, ...user } = await response.json();
+
+storage.save("accessToken", accessToken);
+
+storage.save("profile", user);
+
+alert("You have been logged in successfully");
```
Comment (Spookyrumble) [inline js/api/auth/login.mjs:27]: The error handling can be improved.

I'll just use this Fetch as the example but try to use Try-Catch blocks on your fetches. Something like: 

`export async function login(profile) {
    const loginURL = API_SOCIAL_URL + action;
    console.log(loginURL);

    const body = JSON.stringify(profile);

    const response = await fetch(loginURL, {
    headers: {
        "Content-Type": "application/json"
    },
    method,
    body
})

const { accessToken, ...user } = await response.json();

storage.save("accessToken", accessToken);

storage.save("profile", user);

alert("You have been logged in successfully");`

File: js/handlers/updateProfile.mjs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { getProfile, updateProfile } from "../api/profiles/index.mjs";
+import { load } from "../storage/index.mjs";
+
+
+export async function setUpdateProfileListener() {
+    const form = document.querySelector("#editProfile");
+
+    if (form) {
+        const { name, email } = load("profile");
```
Comment (Spookyrumble) [inline js/handlers/updateProfile.mjs:9]: Nice!

Comment (Spookyrumble) [inline js/handlers/updateProfile.mjs:9]: Nice! Deconstructing an object is something the teachers love to see.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,67 @@
...
+```bash
+npm run dev
+```
+
+## How to use app?
+
+At the landingpage, you need to login or register a new user. For registration you need to have a @noroff.no or @stud.noroff.no email address. After registration, you will be sent to "/posts/" page (homepage). There you can view the posts of the social media app.
+If you want to:
+
+- view singel post: "/post/?id=xxxxx" x= wathever id your post is. if you like to see a post, and nt "null", check out postID: 11795.
```
Comment (Spookyrumble) [inline README.md:50]: Spellcheck ;)

**Comment by HansMariusAndreassen** [comment]
This looks very good. Just keep an eye on the error handling on your fetches, readability and spellchecks. I have left some comments on specific finds, but they apply throughout the files.
Also don't forget to clean up your console.logs and inline comments before delivery.

**Pull request closed without merging**